cypress/e2e/changelog.spec.js

```diff
@@ -3,23 +3,50 @@ const { jirafyChangelog,
     toUpperJiraTickets, addMarkupToChangelog,
     stripBrackets, addCommaSpaceBetweenJiraTickets,
     surroundTicketListWithBrackets } = require('../../utils/changelog')
+const owner = 'onxmaps'
+const repo = 'jirafy-changelog'
 
 describe('Jirafy Changelog', () => {
     context('changelog', () => {
-        it('ensures accurate changelog is generated', () => {
-            cy.exec('./changelog.sh v1.2.0 v1.0.0 onXmaps').then(changelog => changelog.stdout)
-                .then((actualChangelog) => {
-                    cy.fixture('changelog.md').then((expectedChangelog) => {
-                        expect(actualChangelog).to.equal(expectedChangelog)
-                    })
+        it('github api changelog - different tag', () => {
+            cy.request({
+                method: 'POST',
+                url: `https://api.github.com/repos/${owner}/${repo}/releases/generate-notes`,
+                headers: {
+                    'Authorization': `Bearer ${Cypress.env('myToken')}`
+                },
+                body: {
+                    owner: owner,
+                    repo: repo,
+                    tag_name: 'v1.3.0',
+                    target_commitish: 'main',
```

Comment:
Since we know the jirafy-changelog repo uses `main` instead of `master`, hard-coding the target_commitish will work here, but since we didn't hard-code the owner and repo, it might be a good idea to make a constant for the target_commitish for consistency. We could also query the repo to detect whether it uses `master` or `main`.